WordPress/src/main/res/values-night/colors.xml

```diff
@@ -60,4 +60,11 @@
     <color name="reader_image_featured_border">@color/gray_80</color>
     <color name="reader_image_avatar_or_blavatar_border">@color/gray_80</color>
 
+    <!-- Prepublishing Nudges -->
+    <color name="prepublishing_toolbar_icon_color">@color/gray_10</color>
+    <color name="prepublishing_action_type_enabled_color">@color/white</color>
+    <color name="prepublishing_action_result_enabled_color">@color/gray_30</color>
+    <color name="prepublishing_action_type_disabled_color">@color/gray_50</color>
+    <color name="prepublishing_action_result_disabled_color">@color/gray_50</color>
```

Comment:
Instead of using hardcoded different shades of gray, we might want to use emphasis with default colors. I got this feedback for my own changes for reader improvements post card text colors.

Reply:
Thanks @ashiagr could you expound on how I could accomplish this? I haven't done it before. Thank you 🙏

Reply:
I'm still incorporating that feedback and might not be the right person to give an advice. Just an example, we might want to use `on_surface_disabled_selector` for the disabled text that takes `colorOnSurface` as the base color and applies an alpha of .38 `material_emphasis_disabled` to it. It works for both light and dark mode as `colorOnSurface` is different for each mode.

Reply:
Oh, nice! Thanks, @ashiagr this sounds like an awesome solution. I am following the conversation here and I will be looking into this.